pymtl3/passes/PlaceholderConfigs.py

```diff
@@ -0,0 +1,40 @@
+#=========================================================================
+# PlaceholderConfigs.py
+#=========================================================================
+# Author : Peitian Pan
+# Date   : Jul 28, 2019
+"""Configuration of Placeholders."""
+
+import os
+
+from pymtl3.passes.PassConfigs import BasePassConfigs, Checker
+
+
+def expand( v ):
+  return os.path.expanduser(os.path.expandvars(v))
```

Comment:
what is this expand for?

Reply:
It can expand environment variables in the path